src/seedsigner/views/psbt_views.py

```diff
@@ -413,7 +416,19 @@ def run(self):
         if selected_menu_num == RET_CODE__BACK_BUTTON:
             return Destination(BackStackView)
 
-        elif button_data[selected_menu_num] == self.NEXT:
+        elif button_data[selected_menu_num] == self.NEXT or button_data[selected_menu_num] == self.SKIP_VERIFICATION:
+            if button_data[selected_menu_num] == self.SKIP_VERIFICATION:
+                selected_menu_num = self.run_screen(
+                    DireWarningScreen,
+                    title="Security Warning",
+                    status_icon_name=SeedSignerIconConstants.WARNING,
+                    status_headline="Potential Loss of Funds",
```

Comment:
I don't think this needs to be a DireWarningScreen (but a yellow standard `WarningScreen` I'd be good with). This said, I think this PR overall is a good improvement and will merge as is. Would be trivial to change to `WarningScreen` prior to another release if needed.